src/main/java/io/cdap/plugin/ariba/source/connector/AribaConnector.java

```diff
@@ -166,4 +185,62 @@ private JsonArray listTemplates(String pageToken, JsonArray jsonElements) throws
     }
     return jsonElements;
   }
+
+  public List<StructuredRecord> listTemplateData(String templateName) throws IOException, AribaException,
+    InterruptedException {
```

Comment:
where are we throwing interruptedException

Reply:
For the method buildOutputSchema()